Marlin/pins_ZRIB_V20.h

```diff
@@ -0,0 +1,13 @@
+#undef SERVO1_PIN 
+#define RAMPS_D6_PIN ZRIB_V20_D6_PIN 
+#define ZRIB_V20_D6_PIN 6
+#define ZRIB_V20_D9_PIN
+#define RAMPS_D9_PIN ZRIB_V20_D6_PIN
+#undef ORIG_E0_AUTO_FAN_PIN
+#undef ORIG_E2_AUTO_FAN_PIN
+#undef ORIG_E3_AUTO_FAN_PIN
+#define ORIG_E0_AUTO_FAN_PIN ZRIB_V20_D9_PIN
+#define ORIG_E1_AUTO_FAN_PIN ZRIB_V20_D9_PIN
+#define ORIG_E2_AUTO_FAN_PIN ZRIB_V20_D9_PIN
+#define ORIG_E3_AUTO_FAN_PIN ZRIB_V20_D9_PIN
+#include "pins_MKS_13.h"
```

Comment:
Add a header to the top of this pins file (copy and paste the header from `pins_MELZI.h`, for example).